ci/build_docs.sh

```diff
@@ -24,7 +24,6 @@ rapids-mamba-retry install \
     --channel "${PYTHON_CHANNEL}" \
     cucim libcucim
 
-export RAPIDS_VERSION_NUMBER="24.04"
```

Comment:
This wasn't used anywhere. So just removed it

Always possible to add it back whenever we need it again